guides/common/modules/con_services-and-features-supported-in-a-load-balanced-setup.adoc

```diff
@@ -0,0 +1,10 @@
+[id="services-and-features-supported-in-a-load-balanced-setup_{context}"]
+= Services and features supported in a load-balanced setup
+
+A load balancer in {Project} distributes load only for the following services and features:
+
+* Registration using `subscription-manager`
+* Content management with Yum repositories
+* Puppet
+
+You must access other services, such as provisioning, `virt-who`, or remote execution, directly through the individual {SmartProxies} on which these services are running.
```

Comment:
@adamruzicka Based on our conversation, I think nothing about REX is supported for LB. Can you please confirm?

Reply:
tl;dr: all remote execution

REX has its own notion of load balancing. When a job starts, it looks at all the proxies that could be used and picks the least busy one.

For push rex (so push script and ansible), foreman will use the standard strategies to pick one of the available proxies. IIRC there is a load-balancer-aware strategy in katello, but it is behind a setting that is off by default so this strategy is used as a last resort option. Once a proxy is picked, foreman connects to that specific proxy directly and that specific proxy connects to the host directly, none of the opened connections goes through the load balancer.

Pull could be made to work manually if you told the host to connect to a mqtt broker on a specific proxy and then somehow made foreman always pick that particular proxy, thus removing the load balancer from the equation.

Reply:
Based on the explanation above, I think it's okay to keep the wording as-is. @maximiliankolb let me know if you'd prefer me to rephrase this.

Reply:
OK, so maybe the key sentence by Adam is "REX does its own LB". I think it would make sense to somehow mention this. As a user, I'd assume at first that only Puppet _can_ be used to configure hosts when running SPs behind LB.

```suggestion
Other {Project} services, such as provisioning, `virt-who`, or remote execution, go directly through the individual {SmartProxies} on which these services are running.
```
(unsure about "go")

Your call; maybe dropping line 9-10 is best to focus more on what LB _can_ do instead of what it cannot.

Reply:
Thanks a lot for the suggestions Max, that helps.

I need to keep the bit about what LB cannot do, we have a customer request to clarify the limitations of LB.